libs/content/workspace-ffi/src/apple/common_ffi.rs

```diff
@@ -68,8 +68,6 @@ pub unsafe extern "C" fn init_ws(
         surface_height: 0,
     };
 
-    obj.frame();
```

Comment:
I believe this used to exist as a mechanism for initial theme loading so that you weren't flash banged in dark mode by 1 frame of light mode. Could you do a quick sanity check to see if this is truly not needed?

Reply:
(or the opposite)

Reply:
I checked, no flash bang.